Add release notes for v0.1.19 (#1706)
VirtusLab/scala-cli: website/docs/release_notes.md

title: Release notes
sidebar_position: 99
---
import ReactPlayer from 'react-player'

# Release notes

## [v0.1.19](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.19)

## What's Changed

> The Linux Aarch64 native launcher is here! (experimental)

Reviewer: ```suggestion
## The Linux `aarch64` native launcher is here! (experimental)
```

title: Release notes
sidebar_position: 99
---
import ReactPlayer from 'react-player'

# Release notes

## [v0.1.19](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.19)

## What's Changed

> ## Fix `workspace/reload` Intellij IDEA

Reviewer: ```suggestion
## Fix `workspace/reload` for Intellij IDEA
```

> Pick up dependencies from `using` directives after re-importing a BSP project.

Reviewer: ```suggestion
Dependencies (and other configurations) from `using` directives should now always be picked up after a BSP project reload.
```

<ReactPlayer playing controls url='https://user-images.githubusercontent.com/18601388/207319736-534f2d8a-862d-4c0a-8c8a-e52d95ac03e6.mov' />

Fixed by [@Gedochao](https://github.com/Gedochao) in [1681](https://github.com/VirtusLab/scala-cli/pull/1681).

##  `shebang` headers in Markdown

> The `shebang` headers in `scala` code blocks inside a markdown input are always ignored.

Reviewer: NIT I'm not sure if this is worth highlighting... IMO might as well put it in the `Fixes` section (or just leave it as is, I dunno).

```scala
 #!/usr/bin/env -S scala-cli shebang
 println("Hello world")

>  ```

Reviewer: `````suggestion
````markdown
# Scala with `shebang`
A sample code block with the `shebang` header.
```scala
#!/usr/bin/env -S scala-cli shebang
println("Hello world")
```
````
`````
